### DISCUSSION TITLE
Request - option to keep checkboxes enabled

### DISCUSSION THREAD

**Discussion by eddiewebb** [yuin/goldmark#240] [Ideas]
I use goldmark in Hugo and wanted to [generate a checklist](https://webbinaro.dev/blog/adv/checklist/) folks can tick, print and finish.

I was surprised goldmark hardcodes [all checkboxes in tasklist extension as disabled](https://github.com/yuin/goldmark/blob/master/extension/tasklist.go#L92).

I would like an option / format / setting to not do this, and leave off the `disabled` html attribute.

**Why?**
- when printing disabled checkboxes they show and print as greyed out which wastes ink and less attractive
- I want users to "pre-check" anything online before printing, can't do this currently.

Thanks for the great project, curious what you all think?

**Comment by lhunath** [comment]
I second this request. Forcing all checkboxes as disabled and not leaving an option to make them enabled interferes with interactability and scripting.
